src/Controls/src/Core/Window/Window.cs

```diff
@@ -229,10 +230,13 @@ void IWindow.FrameChanged(Rect frame)
 			{
 				SetPropertyChanged(XProperty, nameof(X), x, frame.X);
 				SetPropertyChanged(YProperty, nameof(Y), y, frame.Y);
-				SetPropertyChanged(WidthProperty, nameof(Width), width, frame.Width);
-				SetPropertyChanged(HeightProperty, nameof(Height), height, frame.Height);
 
-				SizeChanged?.Invoke(this, EventArgs.Empty);
+				if (width != frame.Width || height != frame.Height)
+				{
+					SetPropertyChanged(WidthProperty, nameof(Width), width, frame.Width);
```

Comment:
I wonder if there's something similar to what I'm doing here

https://github.com/dotnet/maui/pull/22347/files#diff-5461d9c00748c889dd799eaf32389b419ec8bb26cf1b417496d49d3863d00cc8R1945-R1957

That we should also do. 
Like, if this is being called from the `Handler` we should really propagate these changes back to the handler. Ideally when these properties are set via this path it won't propagate back to the handler right?

Reply:
Not sure if I follow... Are you saying (1 or 2?):

1. This change will prevent the Handler to know when these value changes
2. The previous implementation didn't report to Handler when the value changes and would be good to implement on this PR

The only change I made was to not trigger the SizeChanged event when the Window is moved (X, Y changes), is there a specific event that should be trigger when the Window is moved?

Reply:
Basically it's what's captured in the comment here

https://github.com/dotnet/maui/issues/21306#issuecomment-2007580475

`void IWindow.FrameChanged(Rect frame)` isn't called to move the window. The point of this method is for the platform to inform the xplat code that the window itself has moved. So, basically what happens now is

User changes dims of window interactively => this calls IWindow.FramChanged => This fires MapHeight/MapWidth on handler (This shouldn't happen because we don't need to inform the handler of anything)

or

User programmatically sets Window.Height => this calls [MapHeight](https://github.com/dotnet/maui/blob/0a0be96497d0a9c4ea273b1d5c445eb574846bca/src/Core/src/Handlers/Window/WindowHandler.Windows.cs#L106) => [Which calls UpdateSize on the platform window](https://github.com/dotnet/maui/blob/main/src/Core/src/Platform/Windows/WindowExtensions.cs#L93) => Which will then fire a [window changed event](https://github.com/dotnet/maui/blob/0a0be96497d0a9c4ea273b1d5c445eb574846bca/src/Core/src/Handlers/Window/WindowHandler.Windows.cs#L196) => Which then calls IWindow.SetFrame => Which then calls `MapHeight`